rts/System/FileSystem/ArchiveScanner.cpp

```diff
@@ -561,6 +542,36 @@ void CArchiveScanner::ScanDir(const std::string& curPath, std::deque<std::string
 	}
 }
 
+void CArchiveScanner::ScanArchives(const std::string& curPath, const std::deque<std::string>& foundArchives)
+{
+#ifdef _WIN32
+	static constexpr int NUM_PARALLEL_FILE_READS_SD = 4;
```

Comment:
Could use a rationale for the magic number
```suggestion
	static constexpr int NUM_PARALLEL_FILE_READS_SD = 4; // number chosen because windows logo has 4 colored squares
```